backend/src/main/java/com/soen341/instagram/controller/AccountController.java

```diff
@@ -0,0 +1,51 @@
+package com.soen341.instagram.controller;
+
+import java.util.Date;
+
+// Spring
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+// Project
+import com.soen341.instagram.dto.account.AccountRequestDTO;
+import com.soen341.instagram.service.impl.AccountService;
+
+@RestController
+public class AccountController {
+
+  @Autowired
+  private AccountService accountService;
+
+  @PutMapping(value = "account/passwordModification")
+  public void setPassword(@RequestBody final AccountRequestDTO accountRequest)
```

Comment:
This is not for this review specifically but it could potentially be a good refactoring task in sprint 4 for the team: we should really have some sort of formal convention for our URIs, currently everyone names similar things a little differently.